app/notifications/validators.py

```diff
@@ -137,6 +143,7 @@ def validate_template(template_id, personalisation, service, notification_type):
     template_with_content = create_content_for_notification(template, personalisation)
     if template.template_type == SMS_TYPE:
         check_sms_content_char_count(template_with_content.content_count)
+    check_if_notification_has_any_content(len(template_with_content.__str__()))
```

Comment:
Based on the name of this method, I'd expect to pass it a notification, rather than the length of the notification already. It feels less useful as a standalone method if it is only checking if a number is bigger than 0. I'd suggest changing it so that you call it like:

`check_if_notification_has_any_content(template_with_content)`

and then the method is responsible for calling `__str__` and `len`.

You may also consider changing the name of the method to `check_notification_content_is_not_empty` or `check_notification_has_content`, or `check_notification_has_non_empty_content` as I don't think the word `any` is particularly useful and this method name could be a little more descriptive. 

I think I'd also like to see a `content_count` method added for emails and letters for consistency so actually it might look something more like:

```
def check_notification_content_is_not_empty(template_with_content):
    char_count =template_with_content.content_count()
    if char_count == 0:
        message = 'This message is empty.'
```